src/ai_commands.js

```diff
@@ -161,79 +234,261 @@ const aWrapperHotkey = (() => {
   };
 })();
 
-const aiOutCommon = window.createOutputChannel('AutoIt', 'vscode-autoit-output');
+const aiOutCommon = window.createOutputChannel('AutoIt (global)', 'vscode-autoit-output');
+
+const AiOut = ({ id, aiOutProcess }) => {
+  let prevLine = "",
+      prevLineTimer,
+      isNewLineProcess = true;
+
+  const spacer = " ", //using U+00A0 NO-BREAK SPACE character to avoid white-space character highlight.
+    prefixId = "#" + id + ":" + spacer,
+    prefixEmpty = "".padStart(prefixId.length, spacer),
+    hotkeyFailedMsg = /!!?>Failed Setting Hotkey\(s\)(?::|...)[\r\n]*|(?:false)?--> SetHotKey (?:\(\) )?Restart failed(?:,|. -->) SetHotKey (?:\(\) )?Stop failed\.[\r\n]*/gi,
```

Comment:
@vanowm I now see this every time I run a script --

`--> SetHotKey () Restart failed. --> SetHotKey () Stop failed.`

Reply:
what is the wrapper version? this only been tested with latest version

Reply:
Upgraded to 22.611.2153.13 from [here](https://www.autoitscript.com/autoit3/scite/download/beta_SciTE4AutoIt3/), but problem still exists.

Reply:
Oh, I guess I only tested latest "stable" release...regex doesn't work, because now message received on 2 separate lines
Hopefully #127 fixes it.